base_station/gui/src/components/SAControls.vue

```diff
@@ -119,6 +143,8 @@ export default {
     const esc_0_on = new Toggle(false)
     const esc_1_on = new Toggle(false)
 
+    document.addEventListener('keyup', this.keymonitor)
+
```

Comment:
This looks like duplicate code from `ArmControls.vue`. Again, not familiar with JS/GUI design, but could a separate component be created for the Gimbal controls so we don't have to duplicate this?